modules/accredited_representative_portal/app/models/accredited_representative_portal/power_of_attorney_form.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class PowerOfAttorneyForm < ApplicationRecord
+    belongs_to :power_of_attorney_request,
+               class_name: 'AccreditedRepresentativePortal::PowerOfAttorneyRequest',
+               inverse_of: :form
+
+    has_kms_key
+
+    has_encrypted :data, key: :kms_key, **lockbox_options
+
+    blind_index :city
+    blind_index :state
+    blind_index :zipcode
+
+    # Validations
+    validates :power_of_attorney_request_id, uniqueness: true
```

Comment:
See [discussion](https://api.rubyonrails.org/classes/ActiveRecord/Validations/ClassMethods.html#method-i-validates_uniqueness_of) on race conditions when checking uniqueness from ActiveRecord. Since this uniqueness constraint is not something a user can cause by their input, I don't think there is added benefit to implementing this with validations (which are convenient because they can collect issues for feedback for the user, but that's not our situation).  Also, the PG adapter surfaces the `RecordNotUnique` exception, so we'll know what's up.